backend/src/server/lib/schemas.ts

```diff
@@ -21,3 +21,10 @@ export const slugSchema = ({ min = 1, max = 32, field = "Slug" }: SlugSchemaInpu
       message: `${field} field can only contain lowercase letters, numbers, and hyphens`
     });
 };
+
+export const OrganizationNameSchema = z
```

Comment:
Should we rename this to be a general naming schema and use it everywhere as needed?

I see more places this would apply